django/utils/timezone.py

```diff
@@ -295,13 +295,15 @@ def template_localtime(value, use_tz=None):
 
 # Utilities
 
-def localtime(value, timezone=None):
+def localtime(value=None, timezone=None):
```

Comment:
So I will hold on till @aaugustin opinion or tomorrow morning (timezones difference ;), but it seems to be a nice idea that fallows the parallelization.